tests/movie_test.py

```diff
@@ -307,6 +308,34 @@ def sha256(x):
     # delete the analysis engine
     db.delete_analysis_engine(engine_name=engine_name)
 
+def test_api_track_frame(new_user):
+    """test track_frame """
+    cfg = new_user
+    api_key = cfg[API_KEY]
+    
+    cap = cv2.VideoCapture('tests/data/2019-07-12 circumnutation.mp4')
+    ret, photo0 = cap.read()
+    ret, photo1 = cap.read()
+
```

Comment:
`photo0` should be read from `tests/data/frame_0001.jpg`
`photo1` should be read from `tests/data/frame_0002.jpg`